README.rst

```diff
@@ -1,6 +1,58 @@
-edx Brand Assets
-================
+@edx/brand
+==========
 
 This project contains branding assets and themes for edx.org.
 
-Note: This project should avoid becoming structured as a node module and should have as few dependencies as possible.
+-----
+Usage
+-----
+
+Install this package:
```

Comment:
As we've started discussing... we ultimately need a better story for how this gets included in MFEs.  For now, do you envision that we'll use our npm aliases mechanism to replace an empty Open edX brand with the edx-specific one?

Reply:
Yes I'm working on an OEP that outlines this.